internal/client/client_interceptor.go

```diff
@@ -34,10 +36,54 @@ func NewInterceptedClient(client Client) Client {
 }
 
 func (r interceptedClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.Request, timeout time.Duration) (*tikvrpc.Response, error) {
+	// Build the resource control interceptor if there is one and the resource group name is given.
+	var rcInterceptor interceptor.RPCInterceptor
+	resourceGroupName := req.GetResourceGroupName()
+	if resourceControlInterceptor != nil && len(resourceGroupName) > 0 {
```

Comment:
Should resourceGroupName = "" and "default" result in the same behavior?

Reply:
Fixed. PTAL.